config/routes.rb

```diff
@@ -1,15 +1,18 @@
 Rails.application.routes.draw do
   get 'cart_items/create'
 
-   root 'welcome#index'
+  root 'welcome#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.